includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -3129,6 +3129,101 @@ static function ( $attribute_name ) {
 				}
 				return $title;
 
+			case AMP_Tag_And_Attribute_Sanitizer::DISALLOWED_CHILD_TAG:
+				/* translators: %1$s is the child tag, %2$s is node name */
+				return sprintf(
+					__( "Tag '%1\$s' is disallowed as child of tag '%2\$s'", 'amp' ),
```

Comment:
Two alternative suggestions:

1. Use curly quotes instead of straight ones, as this is WordPress core standard.
2. Alternatively, remove the quotes altogether and instead wrap the `esc_html()` calls below with `code` tags. 

Using `code` throughout these error messages I think will lead to better results.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4405/commits/29eea3c31ad668d83671d2159c931caf86a0a0b3.